configs/components/_base-ruby-augeas.rb

```diff
@@ -17,9 +17,13 @@
 pkg.build_requires "ruby-#{ruby_version}"
 pkg.build_requires "augeas"
 
-pkg.environment "PATH", "$(PATH):/opt/pl-build-tools/bin:/usr/local/bin:/opt/csw/bin:/usr/ccs/bin:/usr/sfw/bin"
+if settings[:use_system_build_tools]
```

Comment:
there shouldn't really be a need to specify different PATHs when using system tools.

We should just leave the PATH alone and if the pl-build-tools dir isn't there nothing will care anyway